### ISSUE TITLE
[Bug] MalformedJsonException exception with valid details.json file

### ISSUE THREAD

**Issue by darthmooguy** [inorichi/tachiyomi#3097]
### Device information
* Tachiyomi version: 0.9.2
* Android version: 6.0
* Device: Acer B3-A30

## Steps to reproduce
1. Create a local manga with the following details.json:
```json
{
  "title": "Berserk",
  "author": "Miura, Kentarou",
  "artist": "Miura, Kentarou",
  "description": "Guts, a former mercenary now known as the \"Black Swordsman,\" is out for revenge. After a tumultuous childhood, he finally finds someone he respects and believes he can trust, only to have everything fall apart when this person takes away everything important to Guts for the purpose of fulfilling his own desires. Now marked for death, Guts becomes condemned to a fate in which he is relentlessly pursued by demonic beings.\nSetting out on a dreadful quest riddled with misfortune, Guts, armed with a massive sword and monstrous strength, will let nothing stop him, not even death itself, until he is finally able to take the head of the one who stripped him—and his loved one—of their humanity.",
  "genre": ["Action", "Adventure", "Demons", "Drama", "Fantasy", "Horror", "Supernatural", "Military", "Psychological", "Seinen"],
  "status": "1",
  "_status values": ["0 = Unknown", "1 = Ongoing", "2 = Completed", "3 = Licensed"]
}
```
2. Open the local manga's info tab.

### Expected behavior
The entered manga details should be shown.

### Actual behavior
There is a toast about a MalformedJsonException and the information is not parsed. If I remove the `"_status values"` section, the file is successfully parsed and the data is saved to the local db.

**Comment by ChrisJAllan** [comment]
I could not reproduce this by copy-pasting the text, could you attach the file that doesn't work?

As a side note, the `"_status values"` is a comment for your reference and is not used by the app in any way, and it's name can be changed if that would help.

**Comment by darthmooguy** [comment]
Hello! Here is the file causing the exception.
[details.zip](https://github.com/inorichi/tachiyomi/files/4635721/details.zip)

**Comment by darthmooguy** [comment]
I built the app from the `dev` branch, and you were right, eb3fff6 fixes the issue! I no longer get the exception, and the information is parsed correctly.

Thanks!

**Comment by arkon** [comment]
Wait, really? I was still encountering issues but I wasn't sure why.

**Comment by darthmooguy** [comment]
Well, on my device at least, the 0.9.2 build from the website had the exception, but not the `dev` build, with the same file.

**Comment by darthmooguy** [comment]
The problem really seemed to be with the `_status values` though, specifically with the values inside the array... If I put it on one line it didn't like the "2 = Completed" part, if I formatted it on multiple lines it didn't like the "0 = Unknown" part... But your change with the eb3fff6 commit seems to have fixed it for my device.

I made a build from the v0.9.2 tag on my Pixel 3A (Android 10) and tested with the same details.json, no error.

Maybe it's a bug related to Android 6?
